samples/bluetooth/central_dfu_smp/sample.yaml

```diff
@@ -3,12 +3,12 @@ sample:
   description: Bluetooth Low Energy Central Device Firmware Update sample
 tests:
   samples.bluetooth.central_dfu_smp:
-    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
+    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
     harness: bluetooth
     tags: bluetooth
   samples.bluetooth.central_dfu_smp.build:
     build_only: true
     build_on_all: true
-    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
+    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
```

Comment:
Should this line be set to the same platforms that we specify in the Readme? But maybe I just fully understand what this file is used for.

Reply:
Just wondering the same as Jan. Should the list in yaml match the table in the README? Now, they don't match.

Reply:
For now, this information in the yaml file are just used for deciding which platforms should be used for testing, but don't know it is still used. yaml refactoring will be done as part of: https://projecttools.nordicsemi.no/jira/browse/NCSDK-8508

Reply:
I see. Probably fine then. It would be nice to test at least the building of the samples in CI.